extensions/ql-vscode/src/common/logging/logger.ts

```diff
@@ -1,23 +1,34 @@
 export interface LogOptions {
-  /** If false, don't output a trailing newline for the log entry. Default true. */
+  // If false, don't output a trailing newline for the log entry. Default true.
   trailingNewline?: boolean;
 
-  /** If specified, add this log entry to the log file at the specified location. */
+  // If specified, add this log entry to the log file at the specified location.
   additionalLogLocation?: string;
 }
 
 export interface Logger {
-  /** Writes the given log message, optionally followed by a newline. */
+  /**
+   * Writes the given log message, optionally followed by a newline.
+   * This function is asynchronous and will only resolve once the message is written
+   * to the side log (if required). It is not necessary to await the results of this
```

Comment:
Possible stupid question: What's the "side log"?

Reply:
Not a stupid question - I'm not sure either, I just copied the comment from elsewhere.